app/models/calculation.rb

```diff
@@ -0,0 +1,23 @@
+class Calculation
+  include ActiveModel::Model
+
+  attr_accessor :left, :right, :operator
+
+  def self.operators
+    [:+, :-, :*, :/]
+  end
+
+  validates :left, numericality: true
+  validates :right, numericality: true
+  validates_inclusion_of :operator, in: operators, message: 'Operator is not operation symbol'
```

Comment:
Rails/Validation: Prefer the new style validations `validates :column, inclusion: value` over `validates_inclusion_of`.